armory/data/datasets.py

```diff
@@ -1355,6 +1355,28 @@ def resisc10(
     )
 
 
+class ClipFrames:
+    """
+    Clip Video Frames
+        Assumes first two dims are (batch, frames, ...)
+    """
+
+    def __init__(self, max_frames):
+        max_frames = int(max_frames)
+        if max_frames <= 0:
+            raise ValueError(f"max_frames {max_frames} must be > 0")
+        self.max_frames = max_frames
+
+    def __call__(self, batch):
```

Comment:
I believe an assumption made here is that a user will always want the _first_ N frames. Is that fair?

Reply:
Correct. If the user wants a more complicated sampling/slicing mechanism, I would rather they set max_frames to None and instead implement the mechanism in their model code.